src/config.rs

```diff
@@ -8,9 +8,14 @@ use rustls::{ClientConfig, ConfigBuilder, WantsVerifier};
 pub trait ConfigBuilderExt {
     /// This configures the platform's trusted certs, as implemented by
     /// rustls-native-certs
+    ///
+    /// This will return None if no valid certs were found. In that case,
```

Comment:
I think this comment fell out-of-date with the impl after the change to use `Result`.

Reply:
It did, sorry. Fixed!